pkg/metrics/prometheus.go

```diff
@@ -4,11 +4,35 @@ import (
 	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"ktbs.dev/teler/common"
 )
 
 // PrometheusInsert logs into metrics
-func PrometheusInsert(data map[string]string) {
+func PrometheusInsert(options *common.Options, data map[string]string) {
 	var counter prometheus.Counter
+	cfg := options.Configs
+
+	//Check if teler have customs rules
+	if cfg.Rules.Threat.Customs != nil {
+		for _, customRule := range cfg.Rules.Threat.Customs {
+			if strings.HasPrefix(data["category"], customRule.Name) {
+				for _, rule := range customRule.Rules {
+					counter = getCustomsRule.WithLabelValues(
+						data["category"],
+						rule.Element,
+						data[rule.Element],
+					)
+
+					counter.Inc()
+
+					//if the rule use "or" operator the metrics will only get the fist match
+					if strings.EqualFold(customRule.Condition, "or") {
+						break
+					}
```

Comment:
teler has a `Selector` field for what elements to display _(if any)_. See [explanations](https://www.notion.so/kitabisa/Configuration-d7c8fab40366406591875bac631bef3f#:~:text=If%20there%20is%20no%20selector%20in%20the%20rules%2C%20then%20the%20first%20matching%20element%20will%20be%20used%20as%20the%20selector.). Example:
https://github.com/kitabisa/teler/blob/39bc58d484c018eba9e45ef04da765cbcf5fce3b/pkg/teler/teler.go#L262-L265